WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -100,6 +103,10 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
         })
     }
 
+    override var topicPostsCount: Int {
```

Comment:
After messing around with the ReaderStream and headers, tags, site topics, tag topics I realize even more the importance of simple naming so could this just be numberOfCards or cardsCount?